kyuubi-server/src/main/resources/sql/sqlite/001-KYUUBI-3967.sqlite.sql

```diff
@@ -0,0 +1,47 @@
+SELECT '< KYUUBI-3967: Shorten column varchar length of metadata table >' AS ' ';
+
+BEGIN;
```

Comment:
Unfortunately, SQLite does not support `ALTER TABLE MODIFY COLUMN`, it was explained at 
https://www3.sqlite.org/matrix/lang_altertable.html

> "Why ALTER TABLE is such a problem for SQLite"